draft-ietf-privacypass-protocol.md

```diff
@@ -488,10 +492,13 @@ to the Architecture document {{I-D.ietf-privacypass-architecture}}.
 
 This document updates the "Token Type" Registry with the following values.
 
-| Value  | Name                   | Publicly Verifiable | Public Metadata | Private Metadata | Nk  | Reference        |
-|:-------|:-----------------------|:--------------------|:----------------|:-----------------|:----|:-----------------|
-| 0x0001 | VOPRF(P-384, SHA-384)  | N                   | N               | N                | 48  | {{private-flow}} |
-| 0x0002 | Blind RSA, 4096        | Y                   | N               | N                | 512 | {{public-flow}}  |
+| Value  | Name                           | Publicly Verifiable | Public Metadata | Private Metadata | Nk  | Reference        |
+|:-------|:-------------------------------|:--------------------|:----------------|:-----------------|:----|:-----------------|
+| 0x0001 | VOPRF (P-384, SHA-384)         | N                   | N               | N                | 48  | {{private-flow}} |
+| 0x0001 | VOPRF (P-521, SHA-512)         | N                   | N               | N                | 64  | {{private-flow}} |
```

Comment:
I think you mean 0x0002 here — but I'd also like to keep 2 as the default blind RSA type